tools/clang/test/DXC/embed_dbg.test

```diff
@@ -2,7 +2,14 @@
 // RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zi /Qembed_debug /Fo %t.embedpdb
 // RUN: %dxc -dumpbin %t.embedpdb | FileCheck  %s --check-prefix=EMBED_PDB
 
-// RUN: ls -1 %T | not grep "\\.pdb$"
+// NOTE: dae4b0d5020646ac43cd4e4924f67eca.pdb is the output filename if exist.
```

Comment:
What is this a hash of?

Reply:
It is based on output bc.
If we change the order of the instructions, the hash will change.